brave/brave-core
# Update pinned roots

Sec review: https://github.com/brave/reviews/issues/1548

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open the following pages:
  - https://go-updater.brave.com
  - https://p3a.brave.com
  - https://variations.brave.com
  - https://creators.brave.com
  - https://mainnet-polygon.brave.com
  - https://anonymous.ads.brave.com
  - https://search.brave.com
  - https://account.brave.com
2. Confirm that none of them return a TLS error.
3. Open https://ssl-pinning.someblog.org/.
4. Confirm that this one returns a `NET::ERR_SSL_PINNED_KEY_NOT_IN_CERT_CHAIN` TLS error:
![Screenshot from 2024-03-11 16-06-45](https://github.com/brave/brave-core/assets/167821/e56352f4-e820-43a7-abb3-b7ae26b4c474)
